tests/reporter/kubernetes.go

```diff
@@ -35,6 +35,24 @@ import (
 	"kubevirt.io/kubevirt/tests/flags"
 )
 
+const (
+	failedCreateDirectory     = "failed to create directory: %v\n"
+	failedOpenFile            = "failed to open the file: %v\n"
+	failedGetVirtHandlerPod   = "failed to get virt-handler pod on node %s: %v\n"
+	virtHandlerName           = "virt-handler"
+	virtLauncherName          = "%s=virt-launcher"
```

Comment:
Confuses me a bit that virtLauncherName has the `%s=` while virtHandlerName doesn't.
In general, I'd suggest name all printf format strings as `*Fmt` (or `*Format`), as you did in other places.